src/main/java/core/basesyntax/service/impl/ReadFileServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReadFileService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReadFileServiceImpl implements ReadFileService {

```

Comment:
```suggestion
public class ReadCsvFileServiceImpl implements ReadCsvFileService {
```
if you add logic inside this method that gonna work only with csv files you should state so in class name